cmd/conduit/root/connectorplugins/connector_plugins.go

```diff
@@ -0,0 +1,43 @@
+// Copyright © 2025 Meroxa, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package connectorplugins
```

Comment:
what about renaming to `connector-plugins`? I saw another package in conduit using the same format `lifecycle-poc`

Reply:
linting wasn't happy about it. need to look

Reply:
@maha-hajja that's part of the path. Can't use the `-` as part of the package name 

```
ST1003: should not use underscores in package names (stylecheck)
```

Reply:
oh thought a dash would be ok, no problem then!